gitops/build/main.yaml

```diff
@@ -96,7 +100,8 @@ stages:
               DOCKER_BUILD_ARGS: ${{ parameters.dockerBuildArgs }}
               ARTIFACT_FILE_NAME: $(artifactFileName)
             displayName: "Build and save image"
-            condition: and(succeeded(), eq(variables['Build.sourceBranch'], '${{ parameters.sourceBranch }}'))
+            ${{ if not(eq(variables['Build.Reason'], 'PullRequest')) }}:
```

Comment:
"If this is not a PullRequest, add this condtion" -> We will always build and save image on PR, is this intended?

Reply:
@ulrikstrid Yes, we need to save it to scan it with trivy. But we never store it as an artifact.

Reply:
@ulrikstrid The CD is triggered by the stage "cd_trigger" being completed.